sky/engine/bindings/dart_runtime_hooks.cc

```diff
@@ -37,6 +39,14 @@ extern void syslog(int, const char *, ...);
 }
 #endif
 
+namespace dart {
+namespace bin {
+
+extern bool capture_stdout;
```

Comment:
Will do.  Just committed the Dart half of this in 1d2d1ce0fcd83d65c68045222157a0fd5b187781.